configs/distributed-training/aws.yaml

```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head_node_type
+  instance_type: m4.2xlarge
+worker_node_types:
+- name: gpu_worker
+  instance_type: g4dn.xlarge
+  min_workers: 1
+  max_workers: 3
```

Comment:
I don't think these values really make sense with the current training script. If we are showing distributed training with 2 GPUs, I think we should either have `min_workers` be 2 (to make the script run immediately) or 1 (if we want to show autoscaling).